### DISCUSSION TITLE
The memory of minio occupies about 1G when there is no load

### DISCUSSION THREAD

**Discussion by polaris-phecda** [minio/minio#15262] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/92513748/178094912-a970519c-6d6e-4581-919f-00d4df05bad1.png)
![image](https://user-images.githubusercontent.com/92513748/178094589-80701eb1-b137-41c3-ab65-d7a688dd9636.png)
When there is no load, the memory of minio occupies about 1G. I wanted to know why, so I exported profile.zip.
Can somebody teach me what is taking up memory? Thanks a lot

**Comment by harshavardhana** [comment, marked as answer]
This is normal - MinIO is not idle its managing your data.

**Reply by polaris-phecda** [reply]
It's kind of you~ thank you very much!  In fact, I just started to learn the source code of minio,  I encountered this problem (only one instance has a high memory usage, and the others are ok). I want to know whether the dataUsageCache here is used to cache data to make users read faster, or to record the data status and provide it to Prometheus.

**Reply by klauspost** [reply]
You are either running a quite old version or you have many top-level folders in your buckets.
